css/style.css

```diff
@@ -0,0 +1,722 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@700&family=Playfair+Display:wght@900&display=swap');
+:root {
+  --primary: #0699A6;
+  --secondary:  rgba(158, 140, 108, 0.2);
+  --black:black;
+  --white: #fff;
+  --gold: #9E8C6C;
+  --font-narrow: 'PT Sans Narrow';
```

Comment:
Nice work moving the fonts to variables. Instead of calling this variable `--font-narrow`, maybe you'd rather call it something more generic like `--font-primary`. This is because, in future, if other developers want to modify this code using some custom code, they can override `--font-primary` and that sounds much better than trying to override `--font-narrow`